.github/workflows/CI.yml

```diff
@@ -74,7 +74,7 @@ jobs:
         shell: bash -e -x -l {0}
         run: |
             cd lfortran
-            ./run_tests.py --skip-run-with-dbg --no-llvm --skip-cpptranslate
+            ./run_tests.py --skip-cpptranslate
```

Comment:
In LFortran, there is one reference test that uses the `run_with_debug` (`-g --debug-with-line-column`) at the following location

https://github.com/lfortran/lfortran/blob/591d78857ddc525ffbaf1d30a671b7f312db4dc8/tests/tests.toml#L3623-L3625

@certik Is the above test sufficient enough for testing the `-g --debug-with-line-column` or should we add another test (hosted in this repository) for testing it?

Reply:
I think that should be sufficient. However you need to disable LLVM tests, or install exactly the same LLVM version that we use, which currently is LLVM 11.

Reply:
Actually, it's the LLVM that is used to build lfortran. And that is not LLVM 11, so we can't run the LLVM tests they will fail.

We should only run the debug tests, not LLVM tests.

Reply:
Can you revert this change? Let's just upgrade LFortran, ensure all old tests work.

As a separate PR let's try to test the debug info.

Reply:
Sure, reverted.

Reply:
>Actually, it's the LLVM that is used to build lfortran. And that is not LLVM 11, so we can't run the LLVM tests they will fail.

Yes, I understand. Thanks. We use `llvm 11.0.1` for updating reference tests and in the lfortran-feedstock we use `llvm 16.0.6`. The reference outputs generated by both the versions do not match.